src/Tribe/Views/V2/View.php

```diff
@@ -611,11 +617,19 @@ public function setup_the_loop( array $args = [] ) {
 			'$_SERVER' => isset( $_SERVER ) ? $_SERVER : [],
 		];
 
-		$this->repository->by_args( wp_parse_args( $args, $this->repository_args ) );
+		$args = wp_parse_args( $args, $this->repository_args );
+
+		$this->repository->by_args( $args );
 
 		$this->set_url( $args, true );
 
-		$wp_query = $this->repository->get_query();
+		/**
+		 * Problematic replacement as context relies on that to have access to the variables
+		 * in the global context, which creates a hard problem to do navigation.
+		 *
+		 * @todo  have conversation with @lucatume about this
```

Comment:
:warning: Comment refers to a TODO task

<sub>hash: 47cfdd07c499e65200c05b231fe407a6</sub>